frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -347,7 +347,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
       <Row>
         <Col span={24}>
           {this.getEnableDisableLayerSwitch(isDisabled, onChange)}
-          <span style={{ fontWeight: 700 }}>
+          <span style={{ fontWeight: 700, wordWrap: "break-word" }}>
```

Comment:
I think this should be a semicolon:
```suggestion
          <span style={{ fontWeight: 700; wordWrap: "break-word" }}>
```

Reply:
The comma is correct. `{ fontWeight: 700; wordWrap: "break-word" }` is a JS object where properties are separated by a comma :nerd_face:

Reply:
Ah, nevermind, I guess this is a react-component and the inner braces are just an object, so comma it is.

Reply:
yep, hence also wordWrap instead of word-wrap, which would have been the pure css syntax :) thanks you two for having a look :heart: